tests/net/dhcpv4/client/src/main.c

```diff
@@ -36,109 +36,65 @@ LOG_MODULE_REGISTER(net_test, CONFIG_NET_DHCPV4_LOG_LEVEL);
 #define NET_LOG_ENABLED 1
 #include "net_private.h"
 
-/* Sample DHCP offer (382 bytes) */
-static const unsigned char offer[382] = {
-0x02, 0x01, 0x06, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x0a, 0xed, 0x48, 0x9e, 0x0a, 0xb8,
-0x09, 0x01, 0x0a, 0xed, 0x48, 0x02, 0x00, 0x00,
-0x5E, 0x00, 0x53, 0x01, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x63, 0x82,
-0x53, 0x63, 0x35, 0x01, 0x02, 0x01, 0x04, 0xff,
-0xff, 0xff, 0x00, 0x3a, 0x04, 0x00, 0x00, 0x54,
-0x60, 0x3b, 0x04, 0x00, 0x00, 0x93, 0xa8, 0x33,
-0x04, 0x00, 0x00, 0xa8, 0xc0, 0x36, 0x04, 0x0a,
-0xb8, 0x09, 0x01, 0x03, 0x04, 0x0a, 0xed, 0x48,
-0x01, 0x0f, 0x0d, 0x66, 0x69, 0x2e, 0x69, 0x6e,
-0x74, 0x65, 0x6c, 0x2e, 0x63, 0x6f, 0x6d, 0x00,
-0x06, 0x0c, 0x0a, 0xf8, 0x02, 0x01, 0xa3, 0x21,
-0xfd, 0x44, 0x0a, 0xb8, 0x09, 0x01, 0x77, 0x3d,
-0x02, 0x66, 0x69, 0x05, 0x69, 0x6e, 0x74, 0x65,
-0x6c, 0x03, 0x63, 0x6f, 0x6d, 0x00, 0x03, 0x67,
-0x65, 0x72, 0x04, 0x63, 0x6f, 0x72, 0x70, 0x05,
-0x69, 0x6e, 0x74, 0x65, 0x6c, 0x03, 0x63, 0x6f,
-0x6d, 0x00, 0x04, 0x63, 0x6f, 0x72, 0x70, 0x05,
-0x69, 0x6e, 0x74, 0x65, 0x6c, 0x03, 0x63, 0x6f,
-0x6d, 0x00, 0x05, 0x69, 0x6e, 0x74, 0x65, 0x6c,
-0x03, 0x63, 0x6f, 0x6d, 0x00, 0x2c, 0x08, 0xa3,
-0x21, 0x07, 0x56, 0x8f, 0xb6, 0xfa, 0x69, 0xff
-};
+/* Sample DHCP offer (394 bytes) */
+static const unsigned char offer[394] = {
+	0x02, 0x01, 0x06, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```

Comment:
I feel like the original formatting with 8 bytes per line made more sense in this context. Also, having only 6 entries in the first line was very likely intentional. At the moment, I'm too lazy to check the specs regarding "why", though.